Cython/Distutils/build_ext.py

```diff
@@ -110,6 +110,10 @@ def build_extension(self, ext):
         if self.get_extension_attr(ext, 'cython_cplus'):
             ext.language = 'c++'
 
+        if hasattr(ext, 'no_c_in_traceback'):
+            c_line_in_traceback = not ext.no_c_in_traceback
+        else:
+            c_line_in_traceback = None
```

Comment:
Looking closer our distutils extensions also mess with the value. I've changed this one but not `old_build_ext`